src/ucp/wireup/address.c

```diff
@@ -54,7 +54,7 @@ typedef struct {
     ucp_rsc_index_t  tl_count;
     unsigned         num_paths;
     size_t           tl_addrs_size;
-} ucp_address_packed_device_t;
+} UCS_S_PACKED ucp_address_packed_device_t;
```

Comment:
why needed? this struct is not passed on the wire
maybe rename to ucp_address_**un**packed_device_t to make it obvious...

Reply:
> why needed? this struct is not passed on the wire

yes, this is my bad. reverted

> maybe rename to ucp_address_**un**packed_device_t to make it obvious...

it is used for both packing/unpacking. I'd leave it as is